index.js

```diff
@@ -0,0 +1,42 @@
+const express = require("express");
+const bodyParser = require("body-parser");
+const cors = require("cors");
+const mongoose = require("mongoose");
+
+const { router } = require("./routes");
+const { initializeDBConnection } = require("./db/db.connect.js");
+const PORT = process.env.PORT || 5000;
+require("dotenv").config();
+
+const app = express();
+
+app.use(bodyParser.json());
```

Comment:
we can use `express.json()` in our code so we don't have to use body-parser :)

i.e

```
app.use(express.json())
```